Dockerfile

```diff
@@ -3,15 +3,17 @@ FROM quay.io/water-hole/ansible-operator
 USER root
 
 RUN curl -Lf http://storage.googleapis.com/kubernetes-release/release/v1.10.7/bin/linux/amd64/kubectl -o /usr/bin/kubectl
+RUN curl -Lf https://apb-oc.s3.amazonaws.com/apb-oc/oc-linux-64bit.tar.gz | tar xzv --directory /usr/bin --wildcards "*/oc" --strip-components=1
```

Comment:
do we need both  oc and kubectl?

Reply:
Yes, due to https://github.com/kubevirt/kubevirt-ansible/blob/master/roles/cdi/tasks/provision.yml#L62